.github/workflows/build.yml

```
@@ -0,0 +1,52 @@
+name: Build
+
+on: [push]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    container: zephyrprojectrtos/ci:latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Cache Zephyr
+        uses: actions/cache@v2
+        with:
+          path: |
+            ../bootloader
+            ../modules
+            ../tools
+            ../zephyr
+          key: ${{ hashFiles('west.yml') }}
+
+      - name: Initialize
+        run: |
+          pip3 install -U west
+          west init -l .
+          west update
+          pip3 install -r ../zephyr/scripts/requirements-base.txt
+          pip3 install -r docs/requirements.txt
+          sudo apt install -y doxygen
+
+      - name: Build firmware
+        run: |
+          west build -b custom_plank -s app
+
+      - name: Archive firmware
+        uses: actions/upload-artifact@v2
+        with:
+          name: firmware
+          path: build/zephyr/zephyr.*
+
+      - name: Build documentation
+        run: |
+          cmake -Hdocs -Bbuild_docs
```

Since 3.13 `-H` means help, and `-S` which is now a documented feature should be used instead.
```suggestion
          cmake -Sdocs -Bbuild_docs
```

---

fixed, thanks! I'm learning a lot of CMake stuff in this PR... :-)